src/test/java/org/swisspush/redisques/handler/RedisquesHttpRequestHandlerTest.java

```diff
@@ -1883,4 +1909,255 @@ public void getMonitorInformation(TestContext context) {
 
         async.complete();
     }
+
+    @Test(timeout = 10000L)
+    public void getStatisticsInformation(TestContext context) throws Exception {
+        Async async = context.async();
+        ArrayList<HashMap<String, Object>> response;
+        flushAll();
+
+        // Test no statistics if no queues
+        when().get("/queuing/statistics").then().assertThat().statusCode(200)
+            .body("queues", empty());
+
+        // Check normal send with missing message consumer -> 1 failure immediately
+        eventBusSend(buildEnqueueOperation("stat_a", "item_a_1"), null);
+        JsonObject json = new JsonObject(
+            "{\n" +
+                "  \"queues\": [\n" +
+                "    {\n" +
+                "      \"name\":\"stat_a\",\n" +
+                "      \"size\":1,\n" +
+                "      \"failures\":1,\n" +
+                "      \"backpressureTime\":0,\n" +
+                "      \"slowdownTime\":1\n" +
+                "    }" +
+                "  ]\n" +
+                "}");
+        when().get("/queuing/statistics")
+            .then()
+            .assertThat().statusCode(200)
+            .body(equalTo(json.toString()));
+        flushAll();
+
+        // Test increasing slowDown Time within 5 seconds
+        eventBusSend(buildEnqueueOperation("stat_b", "item_b_1"), null);
+
+        response = when().get("/queuing/statistics")
+            .then()
+            .assertThat().statusCode(200)
+            .extract().path("queues");
+        assert (response.size() == 1);
```

Comment:
Why do we use `assert` here? Shouldn't we use `async.assertWhatever()` methods?

Reply:
What I mean is using the java `assert` keyword versus the testingframework asswertion methods.

AFAIK behavior of java `assert` keyword depends on JVM options while process startup (*cannot find helpful doc right now*). Do we know that those options are set correctly?

Reply:
Will update and use the context.assert. However, both are working without any difference observed up until now.